spec/system/admin/tenants_spec.rb

```diff
@@ -40,6 +40,52 @@
       expect(page).to have_current_path root_path
       expect(page).to have_link "Sign in"
     end
+
+    scenario "Copies the current user as the new tenant administrator" do
+      user = create(:user, password: "secret_password", password_confirmation: "secret_password")
+      create(:administrator, user: user)
+      login_as(user)
+
+      visit new_admin_tenant_path
+
+      expect(page).to have_content "When you create a tenant, your current user"
+
+      fill_in "Name", with: "Earthlings"
+      fill_in "Subdomain", with: "earth"
+      click_button "Create tenant"
+
+      click_link "earth.lvh.me"
+
+      click_link "Sign in"
+      fill_in "Email or username", with: user.username
+      fill_in "Password", with: "secret_password"
+      click_button "Enter"
+
+      expect(page).to have_content "You have been signed in successfully."
+    end
+
+    scenario "does not create admin by default on seeds when tenant is not default tenant" do
```

Comment:
It took me a while to realize what we were testing. 

```suggestion
    scenario "does not create the default admin for the new tenant" do
```

Also, since these tests are sooooo slow, do you think it would make sense to join them? Or would it be harder to understand the tests?

```ruby
      click_link "Sign in"
      fill_in "Email or username", with: "admin@consul.dev"
      fill_in "Password", with: "12345678"
      click_button "Enter"

      expect(page).to have_content "Invalid Email or username or password."
      expect(page).not_to have_content "You have been signed in successfully."

      fill_in "Email or username", with: user.username
      fill_in "Password", with: "secret_password"
      click_button "Enter"

      expect(page).to have_content "You have been signed in successfully."
```

Reply:
Finally, we kept only one of the tests.